R/utils-pipe.R

```diff
@@ -0,0 +1,14 @@
+#' Pipe operator
```

Comment:
did you end up using a pipe operator in your code?

Reply:
Yes, I spoke with Tiff and she shared it is OK to use.